geoh5py/objects/object_base.py

```diff
@@ -674,8 +698,9 @@ def validate_entity_type(self, entity_type: ObjectType | None) -> ObjectType:
     @property
     def visual_parameters(self) -> VisualParameters | None:
         """
-        Access the visual parameters of the object.
+        The visual parameters of the object.
         """
+        # todo: it sounds like I could have several visual parameters to object
```

Comment:
GA wouldn't let you, but it's true that we don't have mechanics to prevent adding more...

Reply:
I will add one then